### PR TITLE
feat: typography & colors

### PR DESCRIPTION
**Pull request by bhtibrewal**

### CODE REVIEW

File: src/getting_started/typography.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                <h2>Heading 2</h2>
+                <b>Font-size: 3.2rem</b>
+            </div>
+            <div class="tp-box">
+                <h3>Heading 3</h3>
+                <b>Font-size: 2.4rem</b>
+            </div>
+            <div class="tp-box">
+                <h4>Heading 4</h4>
+                <b>Font-size: 1.6rem</b>
```
Comment (rdthetha) [inline src/getting_started/typography.html]: making classes of h1,h2,h3 ,h4 with respective font-size would be better

File: src/getting_started/typography.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            <p>By default, body has a font-size of 16px </p>
+        </div>
+
+        <div class="component-container heading-container">
+            <p class="body-l">Large Text</p>
+            <p class="body-r">Regular Text</p>
+            <p class="body-md">Medium Text</p>
+            <p class="body-sm">Small Text</p>
+        </div>
+
```
Comment (rdthetha) [inline src/getting_started/typography.html]: greyed and center text are missing. please include them as well!

**Review by parvez-rashid — COMMENTED** [review]
Looks good to merge.

File: src/getting_started/typography.html (hunk 2 above)
Comment (yaswanthmyneni) [reply src/getting_started/typography.html]: as @rdthetha suggested add gray and centered text. The remaining looks good to me.

**Review by rahulk31 — APPROVED** [review]
Code looks good Bhavika, Can be merged. 👍

**Pull request merged**